admin/config.yaml

```diff
@@ -79,6 +79,11 @@ vars:
         type: json
       - name: _unknown
         type: unknown
+      - name: _color
+        type: regex
+        regex: "^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$"
+        msg: "Expecting hex format for color, e.g. #007DCD"
```

Comment:
msg, what does it mean?
In this case I understand, but for me it's too shorten...